projects/ion/src/lib/radio/radio.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ion-radio',
+  templateUrl: './radio.component.html',
+  styleUrls: ['./radio.component.scss'],
+})
+export class RadioComponent {
+  @Input() label?: string;
+  @Input() checked?: boolean = false;
```

Comment:
Isso pode ser visto como uma redundância. Já que ao iniciar com um valor `true` ou `false` tem um tipo boolean atribuido de forma implicita. Também se aplica ao disabled.
```suggestion
  @Input() checked? = false;
```